git-cliff-core/src/remote/mod.rs

```diff
@@ -74,7 +74,13 @@ pub(crate) const MAX_PAGE_SIZE: usize = 100;
 /// Trait for handling the different entries returned from the remote.
 pub trait RemoteEntry {
 	/// Returns the API URL for fetching the entries at the specified page.
-	fn url(project_id: i64, api_url: &str, remote: &Remote, page: i32) -> String;
+	fn url(
+		project_id: i64,
+		api_url: &str,
+		remote: &Remote,
+		head: Option<&str>,
```

Comment:
I'm not sure if `head` is the correct naming here, shouldn't this be `range` instead?

Reply:
It isn't a range on the format `BASE..HEAD`, it is only the `HEAD` since this is what the APIs of all remotes want. The range is converted to a head in the `add_remote_data` function in `changelog.rs`.

Reply:
I see, maybe then rename to `ref` instead of `head`? I feel like `head` is too specific. WDYT?